lua/telescope/config.lua

```diff
@@ -506,6 +506,7 @@ append(
   {
     num_pickers = 1,
     limit_entries = 1000,
+    ignore_empty_prompt = false,
```

Comment:
@Conni2461 - Apologies if I got your comment wrong before, but I set it to `false` to avoid any breaking changes. (Though, I agree that having this as `true` would make more sense, but this might differ for others given it might be used by many the existing way already).

Reply:
yeah but defaults should refect what we think are sensible defaults. so i prefer true, even its a breaking change. we have a release branch for exactly that.